content/installation/ocaml.md

```diff
@@ -6,4 +6,49 @@ implementation: official
 weight: 1125
 ---
 
-Please see the [Cucumber OCaml website](https://github.com/cucumber/cucumber.ml).
+[Cucumber.ml](https://github.com/cucumber/cucumber.ml) is an
+implementation written in and useful for
+[OCaml](http://www.ocaml.org/) projects.
+
+# Prerequisites
+
+There are a few complications when working with
+[Cucumber.ml](https://github.com/cucumber/cucumber.ml). The first and
+the most important is that Cucumber.ml delegates its Gherkin parsing
+to the
+[gherkin-c](https://github.com/cucumber/cucumber/tree/master/gherkin/c)
+implementation using OCaml's foreign function interface. This means
+that, before you can use Cucumber.ml, you will need to compile and
+install the gherkin-c as a shared library on your system. On Linux,
+this means you will need to build and install libgherkin.so in
+/usr/lib and copy the gherkin-c header files into
+/usr/include/gherkin.
+
+Cucumber.ml uses the [Dune build
+system](https://github.com/ocaml/dune) and [Opam package
+manager](https://opam.ocaml.org/) for OCaml. You will need to have
+these installed before you can build Cucumber.ml.
+
+Once you have checked out the code and have the above prerequisites
+installed, all you need to do is:
+
+```
+dune build && dune install
+```
+
+This will install the cucumber package into your local Opam
```

Comment:
`Cucumber` instead of `cucumber` for consistency (please check rest of text also?)

Reply:
I updated this with verbatim text as I was referring to the Opam package name.  By convention, all package names in Opam are lower case.